GamesProject/style.css

```diff
@@ -0,0 +1,47 @@
+
+a{
+    color: var(--color-1, #EF4934);
+    text-decoration-line: underline;
+}
+/* Для майбутньої верстки */
+.content{
+    width: 1175px;
+    height: 479px;
+    margin: 140px 52px 398px 53px;
+}
+
+p{
+color: var(--color-2, #221F1F);
+font-family: Roboto;
+font-size: 21px;
```

Comment:
should be 22px, I believe 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/3048cc49-c6d2-475d-9428-9a8fbdad64bb)